management_vm.tf

```diff
@@ -0,0 +1,280 @@
+locals {
+  function_code_path     = "${path.module}/function-app/code"
+  function_app_code_hash = md5(join("", [for f in fileset(local.function_code_path, "**") : filemd5("${local.function_code_path}/${f}")]))
+  function_bins_dir      = "${path.module}/.tf-function-app"
+  storage_account        = azurerm_storage_account.management_sa.name
+  container_name         = azurerm_storage_container.binaries_container.name
+  os                     = "darwin"
+}
+
+resource "null_resource" "upload_function_app" {
```

Comment:
it isn't function-app anymore, the directory name should be `management-vm` or something similar